systemtests/util_test.go

```diff
@@ -158,9 +163,21 @@ func (s *systemtestSuite) rebootstrap() error {
 	return nil
 }
 
+func getDriver() string {
+	driver := "ceph"
+	if strings.Trim(os.Getenv("USE_DRIVER"), " ") != "" {
```

Comment:
ok, sorry to throw this back at you AGAIN. Why are we not using `strings.TrimSpace` here?